actions/hv.patch.reboot.yaml

```diff
@@ -0,0 +1,28 @@
+---
+description: Patch and Reboot a hypervisor
+enabled: true
+entry_point: src/openstack_actions.py
+name: hv.patch.reboot
+parameters:
+  name:
+    type: string
+    description: Name of host in Icinga
+    required: true
+  lib_entry_point:
+    type: string
+    default: workflows.hv_patch_and_reboot.patch_and_reboot
+    immutable: true
+  icinga_account_name:
+    type: string
+    description: "Name of Icinga Account to use. Must be configured in in the pack settings."
+    required: true
+    default: "default"
+  host:
```

Comment:
I think `hypervisor_name` may be better here, we can use this for both icinga and ssh connection, and also matches other workflows